browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -125,7 +125,7 @@ void EphemeralStorageTabHelper::CreateEphemeralStorageAreasForDomainAndURL(
           : base::nullopt);
 
   tld_ephemeral_lifetime_ = content::TLDEphemeralLifetime::GetOrCreate(
-      browser_context, partition, new_domain);
+      browser_context, partition, std::move(new_domain));
```

Comment:
why std::move instead of const ref in `GetOrCreate`

Reply:
replaced with ```const&```